mission_systems/shooter/nodes/Sabertooth2x12.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
I'm glad you're using python for this - however I would take a look at other python scripts (or look at the [PEP8](https://www.python.org/dev/peps/pep-0008/) guidlines) in the navigator or sub repo to get an idea of how things should be formatted. Alternatively you could get a linter that will look at your code as your write it and show you were the PEP errors are.

A couple things right off the bat, I think your python scripts could use more whitespace. PEP8 calls for a new line after every class function. Next, __init__() generally goes at the top of a class so anyone looking at your code knows exactly what the class takes as parameters and knows where to start.